lib/buffer.js

```diff
@@ -105,6 +105,10 @@ function fromString(string, encoding) {
     encoding = 'utf8';
 
   var length = byteLength(string, encoding);
+
+  if (length === 0)
+    return new Buffer(0);
```

Comment:
I do, too. I think there's even a linter rule in place for it. Are you fine with it @trevnorris?